src/DataOnev2/DataOnev2.jl

```diff
@@ -0,0 +1,34 @@
+abstract type DataOnev2 <: DataRepo end
+
+export KNB
+include("KNB.jl")
+
+function description(repo::DataOnev2, mainpage)
+    desc = try text_only(matchall(sel"description para", mainpage.root)) catch "" end
+    authors = try [text_only(i) for i in matchall(sel"individualName", mainpage.root) if contains(string(i.parent), "creator")] catch [] end
+    author = format_authors(authors)
+    license = try text_only(first(matchall(sel"intellectualRights", mainpage.root))) catch "" end
+    rawdate = try Dates.Date((text_only(first(matchall(sel"pubDate", mainpage.root)))), "yyyy-mm-dd") catch "" end
+    year = try Dates.year(rawdate) catch "" end
+    date = try Dates.format(rawdate, "U d, yyyy") catch "" end
+    references = ""
```

Comment:
Please no single line `try`, `catch` blocks.
And especially no catch all conditions.
So called Pokemon-Exception handleing
http://wiki.c2.com/?PokemonExceptionHandling


try block should should only really be used with the following pattern:



```
try
    # code
catch err
   if err isa SpecificExceptionType # specify exception type here
       # code to handle exception
   else
         rethrow()
   end
end
```

It is much better to know what if failing and why,
and then handle it correctly.
than it is to catch everything and produce a result that is silently wrong without explanations.

In general one also would want to spit out warning is a lot of these failures even if they can be handled.

Reply:
I knew this was coming for writing bad code :/. The only issue is that, this being at an abstraction layer, some of the repo APIs don't have the tags required. I felt `try` was inevitable, but sure I'll catch the exceptions in a better way. Keeping into account your previous comment about files with single type, I think to avoid `try`, I need to write some logic (which removes the tags not present) in those files.

Reply:
maybe something like `author(repo, page)`, and `data(repo, page)`